emojivoto/emojivoto.yml

```diff
@@ -165,7 +165,7 @@ spec:
         - emojivoto-vote-bot
         env:
         - name: WEB_HOST
-          value: web-svc:80
+          value: web-svc.emojivoto:80
```

Comment:
what's the background behind this change?

Reply:
the other services address web service like this, so mostly consistency

Reply:
This was my request. If you look [here](https://github.com/runconduit/conduit-examples/blob/master/emojivoto/emojivoto.yml#L117), you can see that the web service addresses the emoji and voting services as `<svc>.<namespace>:<port>`, but the vote bot omits the namespace when addressing the web service. I just wanted them to be consistent across the file. Would also be ok with removing the namespace from the other two hosts, if that's your preference.

Reply:
Ok, this is fine.  I just wanted to make sure I wasn't missing a technical reason for this change.  👍